x/marker/types/msg_test.go

```diff
@@ -245,3 +245,18 @@ func TestMsgAddFinalizeActivateMarkerRequestValidateBasic(t *testing.T) {
 		})
 	}
 }
+
+func TestMsgSupplyIncreaseProposalRequestGetSigners(t *testing.T) {
+	authority := sdk.AccAddress("input111111111111111")
+	targetAddress := sdk.AccAddress("input111111111111111")
+	amount :=
+		sdk.Coin{
+			Amount: math.NewInt(100),
+			Denom:  "",
+		}
+
+	msg := NewMsgSupplyIncreaseProposalRequest(amount, targetAddress.String(), authority.String())
+	res := msg.GetSigners()
+	require.Equal(t, 1, len(res))
+	require.True(t, authority.Equals(res[0]))
```

Comment:
### Thoughts on `.Equal(... len(...))` vs `.Len(...)`

When `.Len(t, res, 1)` fails, the failure message indicates that the differing values are a slice length. It also contains the contents of the slice. This extra info can often help identify problems (e.g. what was unexpectedly added and what was unexpectedly removed). Still, it's best to also include a msg to indicate what slice is being evaluated.

When using `.Equal(t, 1, len(res))`, all the failure message contains is that an expected value was 1, and what the actual value was. There's no context. Including a msg can help with that, but it's clunkier than using `.Len(...)`.

Really, the only time to use `Equal(... len(...))` is when including the values in a failure message has very, very little chance of being helpful.

If you're expecting zero entries, the `.Empty(...)` assertion is the way to go.

### Thoughts on `.True(...)` or `.False(...)`

The ONLY time to ever use this is when you're checking the value of a boolean returned by something being tested. E.g. use it if you're making sure a value is in a map: `val, ok := myMap[foo]; assert.True(t, ok, "is foo in myMap")`. Or e.g. `val, found := k.GetEntry(ctx, 3); assert.True(t, found, "GetEntry(3) found")`.

When a `.True(...)` test fails, the only info in the failure message is that a value was false when it was expected to be true. Such a failure messages is almost never helpful unless there's significant amounts of extra context provided as a msg.

### For this specific case

It's best to define an expected slice, and just pass that slice to `.Equal` with a message indicating what the slice is.
```suggestion
	exp := []sdk.AccAddress{authority}
	require.Equal(t, exp, res, "GetSigners")
```
* Any failure here will indicate that the value came from GetSigners. Even though the name of the test indicates that GetSigners is what's being tested, it's good to have it included in the failure message so that a reader knows for sure that that's the values they're looking at.
* All expected and actual values are included in the failure message.
* Any missing entries are noted in the failure message.
* Any extra entries are noted in the failure message.
* If they're in the wrong order, that's identifiable from the contents of the failure message too.